manual/first_steps_with_root/index.md

```diff
@@ -202,7 +113,7 @@ Calling a function from a ROOT class like `TMath::Pi`:
    3.1415927
 {% endhighlight %}
 
-### Multi-line commands
+#### Multi-line commands
```

Comment:
The multi line command is in reality:
```
{% highlight C++ %}
root [0] {
root (cont'ed, cancel with .@) [1]int j = 0;
root (cont'ed, cancel with .@) [2]for (Int_t i = 0; i < 3; i++){
root (cont'ed, cancel with .@) [3]j = j+i;
root (cont'ed, cancel with .@) [4]cout << "i = " << i << ", j = " << j << endl;
root (cont'ed, cancel with .@) [5]}
root (cont'ed, cancel with .@) [6]}
i = 0, j = 0
i = 1, j = 1
i = 2, j = 3
root [7]
{% endhighlight %}
```

Reply:
For this I'm not sure, it makes it less readable? We even indented the code so that it is easier to read.

Reply:
Yes it makes it less readable but that's what th user will see if he/she tries to enter a macro interactively . I think it is always better to show the reality.

Reply:
It is also possible to indent here.

Reply:
I see your point, I think I will favor readability here though at the expensive of authenticity.